src/ripple/app/misc/impl/LoadFeeTrack.cpp

```diff
@@ -96,21 +96,17 @@ scaleFeeLoad(FeeUnit64 fee, LoadFeeTrack const& feeTrack,
     // compatible. This function is an exception.
     auto lowestTerms = [](auto& a, auto& b)
     {
-        auto value = [](auto val)
+        auto const value = [](auto val)
```

Comment:
I'd remove the `const` from the lambdas. Declaring a lambda `const` doesn't add any restrictions (the lambda's `operator()` is already const`).

Reply:
That's fair. I was thinking that `const` didn't hurt, particularly when I captured it in the second lambda, but it makes sense to remove it, especially if I remove the second lambda.